mllib/src/main/scala/org/apache/spark/mllib/linalg/LAPACK.scala

```diff
@@ -17,9 +17,9 @@
 
 package org.apache.spark.mllib.linalg
 
-import dev.ludovic.netlib.{JavaLAPACK => NetlibJavaLAPACK,
-                           LAPACK => NetlibLAPACK,
-                           NativeLAPACK => NetlibNativeLAPACK}
+import dev.ludovic.netlib.lapack.{JavaLAPACK => NetlibJavaLAPACK,
```

Comment:
The format of `imports` looks strange, I'm not sure if it's standard

Reply:
I will change it to follow the general rules